MLS.Agent.Tests/DocumentationAPITests.cs

```diff
@@ -164,6 +164,48 @@ public async Task Scaffolding_HTML_trydotnet_js_autoEnable_useBlazor_is_true_whe
             }
         }
 
+
+        [Fact]
+        public async Task Scaffolding_HTML_trydotnet_js_autoEnable_useBlazor_is_true_when_package_is_not_specified_and_supports_wasmrunner()
+        {
+            var (name, addSource) = await Create.NupkgWithBlazorEnabled("packageName");
+
+            using (var dir = DisposableDirectory.Create())
+            {
+                var text = $@"
+```cs --package {name}
+```";
+
+                var path = Path.Combine(dir.Directory.FullName, "BlazorTutorial.md");
+                File.WriteAllText(path, text);
+
+                var startupOptions = new StartupOptions(
+                    dir: dir.Directory,
+                    addPackageSource: new WorkspaceServer.PackageSource(addSource.FullName));
+
+                using (var agent = new AgentService(startupOptions))
+                {
+                    var response = await agent.GetAsync(@"/BlazorTutorial.md");
+
+                    response.Should().BeSuccessful();
+
+                    var html = await response.Content.ReadAsStringAsync();
+
+                    var document = new HtmlDocument();
+                    document.LoadHtml(html);
+
+                    var scripts = document.DocumentNode
+                                          .Descendants("body")
+                                          .Single()
+                                          .Descendants("script")
+                                          .Select(s => s.InnerHtml);
+
+                    scripts.Should()
+                           .Contain(s => s.Contains(@"trydotnet.autoEnable({ apiBaseAddress: new URL(""http://localhost""), useBlazor: true });"));
```

Comment:
Do we also want to change this from `useBlazor` to `useWasm`?

Reply:
that is in js api and client space

Reply:
Are you saying we want to use different terminology, or it just for another day because it impacts compatibility?

Reply:
I ma suggesting we do this in another PR as I have to modify Js layer anyway